samples/src/Microsoft.Azure.IIoT.App/src/Pages/_DrawerDiscoverer.razor

```diff
@@ -18,7 +18,7 @@
     <div class="list-group-item text-left">
         <form>
             <div class="form-group">
-                <label>Mask address range (CIDR notation)</label>
+                <label>Scan mask</label>
```

Comment:
The previous string is better, just drop Mask.  Better add help hover if it is unclear what it means.

Reply:
The idea is that it matches the words that are used on the assets page

Reply:
The idea is that it matches the words that are used on the assets page. I changed the other one to "Mask address range"